### DISCUSSION TITLE
save_config_kwargs CLI argument

### DISCUSSION THREAD

**Discussion by nluongo** [Lightning-AI/pytorch-lightning#19812] [General]
I am trying to toggle the overwrite feature of SaveConfigCallback from the command line. I’m guessing that this should be done through the save_config_kwargs argument of LightningCLI, so any help to set values in this dict from the command line would also be appreciated.  Is this possible? 

I’ve attempted the following:
`
MyLightningCLI --config /path/to/my_config.yaml --trainer.devices=1 --trainer.num_nodes=1 --data.num_workers=10 --save_config_kwargs={‘overwrite’:True}`

**Comment by mauvilsa** [comment]
`save_config_kwargs` was not designed to be configurable from command line or config file. This is not possible currently. And not trivial to change to make it possible.
